go/nbs/table_reader.go

```diff
@@ -351,24 +349,26 @@ func canReadAhead(fRec offsetRec, fLength uint32, readStart, readEnd, readAmp, b
 
 	// only consider "wasted" bytes ABOVE tr.BlockSize to be read amplification.
 	fReadAmp := fRec.offset - readEnd
-	if fReadAmp < blockSize {
+	if fReadAmp < tr.blockSize {
```

Comment:
The reason I had `canReadAhead` and `calcReads` take `blockSize` and `ampThresh` as parameters was because I figured we'd want the `frag` tool to be able to report a sort of "max" fragmentation, assuming no amplification and that ANY undesired byte is a wasted byte, as well as fragmentation given a certain block size and allowed amplification.  This change would make it only report the latter. Perhaps that's actually the number we care about?